CHANGELOG.md

```diff
@@ -86,3 +86,9 @@
 
 * Fixed shipping functionality for AfterPay and Klarna Pay Later
 
+** 1.2.3 **
+
+* Replaced 'reason' to 'customer_message' in error output
+* Added receiving of supporting currencies by payment methods from API
```

Comment:
maybe better to write message with more clearly explanation such as :
Added request to retrieve a list of available currencies for every payment method.